python/cuml/cluster/__init__.py

```diff
@@ -16,3 +16,4 @@
 
 from cuml.cluster.dbscan import DBSCAN
 from cuml.cluster.kmeans import KMeans
+from cuml.cluster.agglomerative import AgglomerativeClustering
```

Comment:
Can you add it to the general init too? (as someone benchmarking multiple models in the explainers, I looove the ability of being able to do `from cuml import bla` without needing to know the second level :smile: ) https://github.com/rapidsai/cuml/blob/0883026bf6dd085a4d5afb036df7c3a4989efda2/python/cuml/__init__.py#L22